### DISCUSSION TITLE
GPU docker support

### DISCUSSION THREAD

**Discussion by bfigueir** [OvenMediaLabs/OvenMediaEngine#1249] [Help]
Hi all,

is there any docker image already built with GPU support on docker hub or do we have to always built from the source code?
Also, has anyone used OME in Azure Kubernetes with GPU enabled?

Thanks
Bruno

**Comment by bfigueir** [comment]
@getroot ?

**Comment by getroot** [comment]
Someone else might be able to auto build and deploy. However, the only Dockerhub we officially maintain is https://hub.docker.com/r/airensoft/ovenmediaengine. Here's all we auto-build.
Please don't tag me unless you have a specific reason.
